schema/deploy/computed_columns/form_change_calculated_total_payment_amount_to_date.sql

```diff
@@ -0,0 +1,25 @@
+-- Deploy cif:computed_columns/form_change_calculated_total_payment_amount_to_date from pg
+
+begin;
+
+create or replace function cif.form_change_calculated_total_payment_amount_to_date(cif.form_change)
+returns numeric
+as
+$fn$
+
+select (($1.new_form_data->>'maxFundingAmount')::numeric*.8) +
+  coalesce(
+    ((select new_form_data->>'projectSummaryReportPayment'
+      from cif.form_change
+      where project_revision_id = $1.project_revision_id
+      and json_schema_name = 'project_summary_report'
+      and operation != 'archive'
+      order by id desc
+      limit 1)::numeric),
```

Comment:
This is so we get the most recent one. Not sure why we haven't needed to do this in other calculation computed columns